contracts/ERC20Safe.sol

```diff
@@ -232,7 +232,12 @@ contract ERC20Safe is BridgeRole, Pausable {
      @return a list of deposits included in this batch
     */
     function getDeposits(uint256 batchNonce) public view returns (Deposit[] memory) {
-        return batchDeposits[batchNonce - 1];
+        Batch memory batch = batches[batchNonce - 1];
+        if (_isBatchFinal(batch)) {
+            return batchDeposits[batchNonce - 1];
+        }
+
+        return batchDeposits[type(uint256).max];
```

Comment:
Isn't there a way to return an empty array instead of getting an empty one from an invalid ID?

Reply:
only if we store in the memory an empty array and return that each time